godot-macros/src/class/data_models/field_export.rs

```diff
@@ -519,6 +543,13 @@ impl ExportType {
             Self::ColorNoAlpha => quote_export_func! { export_color_no_alpha() },
         }
     }
+
+    pub fn to_export_usage(&self) -> Option<Ident> {
+        match self {
+            Self::Storage => Some(Ident::new("STORAGE", Span::call_site())),
```

Comment:
We have a utility function for this:
```suggestion
            Self::Storage => Some(ident("STORAGE")),
```